test/extended/router/h2spec.go

```diff
@@ -153,63 +216,43 @@ func failingTests(testSuites []*h2spec.JUnitTestSuite) []h2specFailingTest {
 	return failures
 }
 
-func runConformanceTests(oc *exutil.CLI, t h2specRouteTypeTest) []*h2spec.JUnitTestSuite {
-	podName := "h2spec"
-	e2e.ExpectNoError(e2epod.WaitForPodNameRunningInNamespace(oc.KubeClient(), podName, oc.KubeFramework().Namespace.Name))
-
-	var results []*h2spec.JUnitTestSuite
+func runConformanceTests(oc *exutil.CLI, host, podName string) ([]*h2spec.JUnitTestSuite, error) {
+	g.By("Running the h2spec CLI test")
 
-	o.Expect(wait.PollImmediate(time.Second, 5*time.Minute, func() (bool, error) {
-		g.By("Running the h2spec CLI test")
+	// this is the output file in the pod
+	outputFile := "/tmp/h2spec-results"
```

Comment:
Should the `outputFile` be composed from the `podName`, or passed in as a parameter?

Reply:
Or just hard-coded in the `h2specCommand` func?

Reply:
I would rather not dictate behaviour. "Please tell me where you'd like the output" is the goal of specifying this here.